packages/ember-testing/tests/adapters/qunit_test.js

```diff
@@ -1,51 +1,55 @@
 import { run } from 'ember-metal';
 import QUnitAdapter from '../../adapters/qunit';
+import { moduleFor, AbstractTestCase } from 'internal-test-helpers';
 
 var adapter;
 
-QUnit.module('ember-testing QUnitAdapter', {
-  setup() {
+moduleFor('ember-testing QUnitAdapter', class extends AbstractTestCase {
+  constructor() {
+    super();
     adapter = new QUnitAdapter();
-  },
+  }
+
   teardown() {
     run(adapter, adapter.destroy);
   }
-});
 
-QUnit.test('asyncStart calls stop', function() {
-  var originalStop = QUnit.stop;
-  try {
-    QUnit.stop = function() {
-      ok(true, 'stop called');
-    };
-    adapter.asyncStart();
-  } finally {
-    QUnit.stop = originalStop;
+  ['@test asyncStart calls stop'](assert) {
+    var originalStop = QUnit.stop;
+    try {
+      QUnit.stop = function() {
```

Comment:
I'm not very sure what to do about `QUnit.stop` and `QUnit.start`

Reply:
The places that are explicitly testing `asyncStart` and `asyncStop` should continue to use `QUnit.stop` / `QUnit.start` for now, but anywhere else it is being done should be migrated to `assert.async()`.